config/locales/ar.yml

```diff
@@ -268,7 +268,7 @@ ar:
         help: While you can have multiple certificates, only one can be automatically issued; it will be issued when a student <em>completes</em> a course.
         help_url: https://docs.pupilfirst.com/users/certificates#automatically-issuing-certificates
         label: Should students be automatically issued this certificate?
-      cannot_be_auto_issued_warning: Please note that the last level of this course does not have any milestone targets. This certificate will be auto-issued only if the last level has at least one milestone target.
+      cannot_be_auto_issued_warning: Please note that the last level of this course does not have any milestones. This certificate will be auto-issued only if the last level has at least one milestone.
```

Comment:
Invalid translation, referring to old behaviour. The `en` translation is the correct one; it should be okay to just delete this key.